### PR TITLE
Add a prepare_release.sh script that automates some manual steps.

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

**Review by beets — APPROVED** [review]
nice!!

**Review by shifucun — APPROVED** [review]
Thanks for the update!

File: scripts/prepare_release.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  ./scripts/update_golden_staging.sh
+  if [ $? -ne 0 ]; then
+    echo "ERROR: Failed to update proto"
+    exit 1
+  fi
+}
+
+function commit() {
+  echo ""
+  echo "==== Commiting the change ===="
```
Comment (shifucun) [inline scripts/prepare_release.sh:44]: nit: Committing

Comment (pradh) [reply scripts/prepare_release.sh]: Done

**Pull request merged**